### PR TITLE
story/TYTA-128/create-landing-page

### PR DESCRIPTION
**Pull request by remirubis**
### Description

Creation of a global landing page and editing of some components like the header to change default logo link when we are connected

**Render**

screenshot-localhost_3000-2022 12 10-16_01_09

### CODE REVIEW

File: src/components/Elements/ButtonMailto.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    onClick={(e) => {
+      window.open(`mailto:${email}`, '_blank');
+      e.preventDefault();
+    }}
+  >
+    {children}
+  </Link>
+);
+
+export default ButtonMailto;
```
Comment (FabienHP) [inline src/components/Elements/ButtonMailto.jsx:17]: This suggestion fix the "jsx-a11y/anchor-is-valid" eslint error

```suggestion
import { Box } from '@mui/system';
import React from 'react';

const ButtonMailto = ({ email, children }) => (
  <Box
    sx={{ cursor: 'pointer' }}
    onClick={(e) => {
      window.open(`mailto:${email}`, '_blank');
      e.preventDefault();
    }}
  >
    {children}
  </Box>
);

export default ButtonMailto;
```

Comment (FabienHP) [inline src/components/Elements/ButtonMailto.jsx]: This suggestion fix the "jsx-a11y/anchor-is-valid" eslint error

```suggestion
import { Box } from '@mui/system';
import React from 'react';

const ButtonMailto = ({ email, children }) => (
  <Box
    sx={{ cursor: 'pointer' }}
    onClick={(e) => {
      window.open(`mailto:${email}`, '_blank');
      e.preventDefault();
    }}
  >
    {children}
  </Box>
);

export default ButtonMailto;
```

File: src/pages/Forbidden/Forbidden.jsx [hunk 2]
```diff
@@ -1,47 +1,39 @@
...
+      alt="Lock page."
+      src={ForbiddenImg}
+    />
+    <Box>
+      <Typography variant="h3">Oups...</Typography>
+      <Typography variant="h5">Not authorized</Typography>
+      <Typography variant="body1" sx={{ my: 2, mb: 5 }}>
+        You need to be connected, or you don't have access to this page
+      </Typography>
+      <Button variant="contained" startIcon={<ArrowBackIcon />} href="/">
```
Comment (FabienHP) [inline src/pages/Forbidden/Forbidden.jsx:32]: I dont know is possible in Button api of material, thx
[MaterialUi Button href desc](https://mui.com/material-ui/api/button/#:~:text=of%20its%20container.-,href,-string)

**Review by FabienHP — CHANGES_REQUESTED** [review]

File: src/components/Elements/ButtonMailto.jsx (hunk 1 above)
Comment (remirubis) [reply src/components/Elements/ButtonMailto.jsx]: Done, thanks! ✅

**Review by BrystoQ — APPROVED** [review]
LGTM!

**Review by FabienHP — APPROVED** [review]

**Pull request merged**